autoPyTorch/pipeline/components/training/trainer/AdversarialTrainer.py

```diff
@@ -189,12 +189,17 @@ def get_hyperparameter_search_space(
             default_value=3),
         epsilon: HyperparameterSearchSpace = HyperparameterSearchSpace(
             hyperparameter="epsilon",
-            value_range=(0.05, 0.2),
-            default_value=0.2),
+            value_range=(0.001, 0.15),
+            default_value=0.007,
+            log=True),
     ) -> ConfigurationSpace:
         cs = ConfigurationSpace()
 
+        epsilon = HyperparameterSearchSpace(hyperparameter="epsilon",
+                                            value_range=(0.007, 0.007),
+                                            default_value=0.007)
```

Comment:
Constant value? but then why do we pass epsilon in the argument as well??

Reply:
An argument to replicate the results of the regularization cocktail paper, will not stay when merged to development.

Reply:
should I remove it now?

Reply:
Not at the moment.